--- kivy/core/text/markup.py.orig      2018-04-24 09:38:06.357231117 +0000
+++ kivy/core/text/markup.py   2018-04-24 09:38:19.536283261 +0000
@@ -191,7 +191,7 @@
             elif item[:5] == '[sub]':
                 spush('font_size')
                 spush('script')
-                options['font_size'] = options['font_size'] * .5
+                options['font_size'] = options['font_size'] * .7
                 options['script'] = 'subscript'
             elif item == '[/sub]':
                 spop('font_size')
@@ -199,7 +199,7 @@
             elif item[:5] == '[sup]':
                 spush('font_size')
                 spush('script')
-                options['font_size'] = options['font_size'] * .5
+                options['font_size'] = options['font_size'] * .7
                 options['script'] = 'superscript'
             elif item == '[/sup]':
                 spop('font_size')
